sdk/appconfiguration/tests.yml

```diff
@@ -0,0 +1,8 @@
+trigger: none
+
+jobs:
+  - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+    parameters:
+      ServiceDirectory: appconfiguration
+      EnvVars:
+        AZCONFIG_CONNECTION_STRING: $(java-azconfig-test-connection-string)
```

Comment:
nit: missing blank line at end of file.

Reply:
You old-school hackers and your newlines at the end of files.... fixed :)